Packs/FlashpointFeed/Classifiers/classifier-mapper-incoming-Flashpoint_Feed.json

```diff
@@ -0,0 +1,690 @@
+{
```

Comment:
Currently, in Feed Integrations, we map the indicator fields within the integration code (so we can handle the information that comes back most conveniently, and also save Classifier configuration)
Is there a specific reason you created the Classifier?

Reply:
We have classified the indicator through code (generic and custom indicator) and wanted to have fewer dependencies on the code so we created a mapper to map the indicator fields.

Reply:
If there is no special reason, then please add the mapping of the fields within the code (as in all the feeds we have).